x-pack/plugins/security_solution/public/network/pages/details/index.tsx

```diff
@@ -24,7 +24,7 @@ import { IpOverview } from '../../components/details';
 import { SiemSearchBar } from '../../../common/components/search_bar';
 import { WrapperPage } from '../../../common/components/wrapper_page';
 import { useNetworkDetails } from '../../containers/details';
-import { FlowTargetSourceDest, LastEventIndexKey } from '../../../graphql/types';
+import { FlowTargetSourceDest } from '../../../graphql/types';
```

Comment:
shall we import it from x-pack/plugins/security_solution/common/search_strategy/security_solution/network/common/index.ts